app/views/shared/_footer.html.haml

```diff
@@ -143,7 +143,11 @@
           %div
             = t :footer_legal_data_text
             = t :footer_legal_data_call
-            %a{'cookies-policy-modal'=> true, 'cookies-banner'=> @show_cookies_banner}
+            - if "#{Spree::Config.privacy_policy_url}" != ""
+              %a{href: "#{Spree::Config.privacy_policy_url}"}
```

Comment:
Same here: `%a{href: Spree::Config.privacy_policy_url}` should work.

Reply:
Yes, fixed.